utilities/ride-model-util/src/main/java/com/adobe/ride/utilities/model/ModelObject.java

```diff
@@ -1427,7 +1594,7 @@ public Object generateNodeValue(String parentPath, String key, JSONObject proper
    * @param relativePath relative path to referenced schema
    * @return String
    */
-  protected String getRelativeResourceLocation(String controlPath, String relativePath) {
+  protected static String getRelativeResourceLocation(String controlPath, String relativePath) {
```

Comment:
This is sub-ref'd by a public static method, so this needed to be static as well.